src/model/Model.js

```diff
@@ -245,7 +247,11 @@ export default class Model extends ModelBase {
         const value = json[attr];
 
         if (_.isString(value)) {
-          json[attr] = JSON.parse(value);
+          try {
+            json[attr] = JSON.parse(value);
+          } catch (err) {
+            // json column might contain plain single string which is not wrapped to array / object
+          }
```

Comment:
Looks like it was necessary, because pg driver converts json-string to non quoted string.